config/runtime.exs

```diff
@@ -1,5 +1,22 @@
 import Config
 
+if host = System.get_env("HOST") do
+  config :document_viewer, DocumentViewerWeb.Endpoint,
+    url: [host: host, port: 80],
+    secret_key_base: System.get_env("SECRET_KEY_BASE")
```

Comment:
question: does this part need to be part of the `HOST` if? or could it be separate?

Reply:
Nope, not only could it be separate, it should really have its own `if` statement to be safe. Fixed.

Reply:
I don't think you want it to have an `if` statement: you want the startup to crash if `SECRET_KEY_BASE` isn't set.